src/Draco.Compiler.Tests/Semantics/SymbolResolutionTests.cs

```diff
@@ -1819,7 +1798,10 @@ public static class FooModule{
         // Act
         var compilation = Compilation.Create(
             syntaxTrees: ImmutableArray.Create(main),
-            metadataReferences: ImmutableArray.Create(fooRef));
+            metadataReferences: Basic.Reference.Assemblies.Net70.ReferenceInfos.All
+                .Select(r => MetadataReference.FromPeStream(new MemoryStream(r.ImageBytes)))
+                .Append(fooRef)
+                .ToImmutableArray());
```

Comment:
Why not have CreateCompilation overload with one synax tree and params references?

Reply:
There are other small variations in the other Compilation.Create methods, like specifying the root module path explicitly and such. I didn't want to cover all cases because at that point we'd be close to just copying Compilation.Create